client/asset/kvdb/kvdb.go

```diff
@@ -1,157 +0,0 @@
-// This code is available on the terms of the project LICENSE.md file,
-// also available online at https://blueoakcouncil.org/license/1.0.0.
-
-package kvdb
-
-import (
-	"context"
-	"encoding"
-	"errors"
-	"time"
-
-	"decred.org/dcrdex/dex"
-	"github.com/dgraph-io/badger"
-)
-
-type KeyValueDB interface {
-	Store(k []byte, v encoding.BinaryMarshaler) error
-	Close() error
-	ForEach(f func(k, v []byte) error) error
-	Run(context.Context)
-	Delete(k []byte) error
-}
-
-type kvDB struct {
-	*badger.DB
-	log dex.Logger
-}
-
-func NewFileDB(filePath string, log dex.Logger) (KeyValueDB, error) {
-	// If memory use is a concern, could try
-	//   .WithValueLogLoadingMode(options.FileIO) // default options.MemoryMap
-	//   .WithMaxTableSize(sz int64); // bytes, default 6MB
-	//   .WithValueLogFileSize(sz int64), bytes, default 1 GB, must be 1MB <= sz <= 1GB
-	opts := badger.DefaultOptions(filePath).WithLogger(&badgerLoggerWrapper{log})
-	db, err := badger.Open(opts)
-	if err == badger.ErrTruncateNeeded {
-		// Probably a Windows thing.
-		// https://github.com/dgraph-io/badger/issues/744
-		log.Warnf("NewFileDB badger db: %v", err)
-		// Try again with value log truncation enabled.
-		opts.Truncate = true
-		log.Warnf("Attempting to reopen badger DB with the Truncate option set...")
-		db, err = badger.Open(opts)
-	}
-	if err != nil {
-		return nil, err
-	}
-
-	return &kvDB{db, log}, nil
-}
-
-// Run starts the garbage collection loop.
-func (d *kvDB) Run(ctx context.Context) {
-	ticker := time.NewTicker(5 * time.Minute)
-	defer ticker.Stop()
-	for {
-		select {
-		case <-ticker.C:
-			err := d.RunValueLogGC(0.5)
-			if err != nil && !errors.Is(err, badger.ErrNoRewrite) {
-				d.log.Errorf("garbage collection error: %v", err)
-			}
```

Comment:
What happened to this garbage collection loop?

Reply:
I added it back.